backend/substrapp/tasks/k8s_backend.py

```diff
@@ -90,7 +90,14 @@ def watch_pod(name, watch_init_container=False):
     error = None
     watch_container = not watch_init_container
 
-    logger.error(f'Checking for pod {name} init: {watch_init_container} main: {watch_container}')
+    logger.info(f'Waiting for pod {name}...')
+
+    def get_error(state):
+        error = state.reason
+        if state.message is not None:
+            error += f' ({state.message})'
+        return error
```

Comment:
I think this function can be created outside this one :)

Reply:
https://github.com/SubstraFoundation/substra-backend/pull/314/commits/5c3a741ead9333de4d44a89c3a3bba0e9daec5ad